src/basics/Inputs.js

```diff
@@ -2,10 +2,20 @@ import React from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 
-import { ARROW_SIZES } from "constants/arrows";
 import { FONT_WEIGHT, PALETTE } from "constants/styles";
 
-import { Arrow } from "basics/Icons";
+import Chevron from "assets/icons/chevron.svg";
```

Comment:
Replaced an old arrow with a new chevron

Reply:
It might be nice to keep this import as `import { Arrow } from 'basics/icons'` so we can abstract out the exact source of the icon? I'm not sure if we're using `Arrow` in multiple places though.

Reply:
I'll move `<CloseIcon/>` in `basics/icons` as well